src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -37,12 +39,16 @@ const useSaveAnonChatHook = () => {
     type: string
   ) => {
     if (!localDb) return;
-    const chatName = await getAnonymousChatName(object?.members);
+    const {channelName, channelImage, originalMembers} = await getChannelListInfo(
+      object,
+      signedProfileId,
+      anonProfileId
+    );
 
     const initAnonymousChat: ModifyAnonymousChatData = {
       ...object,
-      targetName: chatName?.name,
-      targetImage: chatName?.image
+      targetName: channelName,
+      targetImage: channelImage
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `getAnonymousChatName` to `getChannelListInfo` seems to be a significant one. It's important to ensure that the new function `getChannelListInfo` is thoroughly tested and behaves as expected in all scenarios, especially considering it now takes additional parameters `signedProfileId` and `anonProfileId`. If these parameters are not always guaranteed to be available or valid, you should add error handling for such cases.



<!-- This is an auto-generated comment by OSS CodeRabbit -->